components/GameResult/GameResult.tsx

```diff
@@ -0,0 +1,131 @@
+'use client';
+import Image from 'next/image';
+import classNames from 'classnames';
+import styles from './GameResult.module.scss';
+import imgWin from 'public/images/win.png';
+import playerImg from 'public/logo/account-avatar.png';
+import playerImg2 from 'public/logo/account-avatar2.png';
+import loseImg from 'public/images/lose.svg';
+import drawImg from 'public/images/draw.svg';
+import { IGameResultProps } from './GameResultProps';
+import { BlockPayedGame, Button, CustomButton } from 'components/shared';
+import { ScoreCard } from 'components/GameResult/ScoreCard/index';
+import { TeamMemberBasic } from 'components/shared/ui/TeamMemberBasic';
+import team from 'data/team.json';
+export const GameResult = (props: IGameResultProps) => {
+  const { result } = props;
+  const resultClass = classNames(styles.container, {
+    [styles.win]: result === 'win',
+    [styles.lose]: result === 'lose',
+    [styles.draw]: result === 'draw',
+  });
+  const player1 = {
+    playerName: '0xh20...7260',
+    playerImg: playerImg,
+    showWinText: result === 'win',
+  };
+  const player2 = {
+    playerName: '0xh07...6035',
+    playerImg: playerImg2,
+    showWinText: result === 'lose',
+  };
+  let content;
+
+  if (result === 'win') {
```

Comment:
Something went completely wrong with the "dry" concept. 
let content may have there different values, but those values have identical structures and differ only in values.